src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -370,6 +384,14 @@ public function getSecurityContext()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getLocale(Request $request)
+    {
+        return;
```

Comment:
Shouldn't this be returning something? Otherwise the security won't apply here, but I think it certainly should (because media is translatable)